site/content/blogs/release-v123.en.md

```diff
@@ -35,7 +35,15 @@ Copilot v1.23 brings several new features and improvements:
 
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
-## App Runner private services
+## App Runner Private Services
+You can now create App Runner private services using Copilot. Simply update your Request-Driven Web Service manifest with:
+```
+http:
+  private: true
+```
+And deploy! Your service is now only reachable by other services in your Copilot environment.
```

Comment:
Would you like to expand over here saying that we'll create the VPC interface endpoint for them by default? Alternatively, if they have an existing interface endpoint for app runner they can bring their own.